README.md

```diff
@@ -18,9 +18,50 @@ The unleash defined strategies are included, to support custom strategies
 use the `ClientBuilder` and call the `strategy` method to register your custom
 strategy memoization function.
 
-The crate documentation should be consulted for more detail.
+The [crate documentation](https://docs.rs/unleash-api-client/0.6.0/unleash_api_client) should be consulted for more detail.
```

Comment:
I think this should link to https://docs.rs/unleash-api-client so folk are taken to the latest release.

Reply:
Done!